### ISSUE TITLE
Integrate LMS server [Feature request]

### ISSUE THREAD

**Issue by Kristmas** [robgeerts/dashticz_v2#92]
Is it possible to have a integration for Logitech Media Server ? What a mean is , larger control buttons for Play / Pause / Stop / Volume/ ... and albumart cover. Maybe it's not possible but a searching for this 4 a long time. A frontpage with good LMS control. Something about jivelite. A found some code from a forum member but a don't know if it's usefull. A always want 2 help but i'm not a coder. Thanks anyway.

**Comment by ultrara1n** [comment]
Will have a look into this, you got a web interface for your LMS right? IP:9000?

**Comment by Kristmas** [comment]
Yes , that's correct , but in my case it's 9002.   A have some files from a other user that's maybe usefull to you ?

**Comment by Kristmas** [comment]
Sorry wrong button.

**Comment by ultrara1n** [comment]
Of course, the more informations would be nice, i've found an unofficial documentation of the interface and just installed the LMS on my PC and will test it out.

**Comment by ultrara1n** [comment]
So if you open IP:9002/status.html there is a minimalistic html interface?
And if you have music playing and opening IP:9002/status.html?p0=stop its stopping?

**Comment by Kristmas** [comment]
That's right.
![status](https://cloud.githubusercontent.com/assets/13322467/26488063/479ab2fc-4202-11e7-8a00-508e6ac08304.JPG)

**Comment by Kristmas** [comment]
Here is the link to the code a found.
[http://www.domoticz.com/forum/download/file.php?id=2911](url)
Maybe you can do something with it.
Thanks 4 helping me out.

**Comment by Kristmas** [comment]
Is there some progress on this issue ?

**Comment by robgeerts** [comment]
@ultrara1n Are you working on this?

**Comment by ultrara1n** [comment]
@robgeerts I had to stop development for the moment because of study exams, feel free to create a solution. I've nothing to share with you, it was only a concept in my mind, unfortunately.
